skyvern-frontend/src/App.tsx

```diff
@@ -1,5 +1,12 @@
+import { Button } from "./components/ui/Button";
+import { ThemeProvider } from "@/components/ThemeProvider";
+
 function App() {
-  return <></>;
+  return (
+    <ThemeProvider defaultTheme="dark" storageKey="skyvern-theme">
+      <Button variant={"destructive"}>Hello Shadcn button!</Button>
```

Comment:
🎉 first you app!

maybe 😃 
```suggestion
      <Button variant={"destructive"}>Hello Skyvern!</Button>
```

Reply:
Great, added.